packages/react-next/src/components/Slider/__snapshots__/Slider.test.tsx.snap

```diff
@@ -239,22 +91,18 @@ exports[`Slider renders correctly 1`] = `
             box-sizing: border-box;
             color: #323130;
             display: block;
-            flex-shrink: 1;
             font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
             font-size: 14px;
             font-weight: 600;
-            line-height: 1;
-            margin-bottom: 0;
-            margin-left: 8px;
-            margin-right: 8px;
-            margin-top: 0;
+            margin-bottom: 0px;
```

Comment:
hmm why is this block of css-in-js  not removed

Reply:
@xugao I believe that this is still here because the underlying component is a label which still has css-in-js and so it would remain. I'm not sure about that though.

Reply:
gotcha, makes sense